content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 3
```

Comment:
```suggestion
weight: 40
```

Reply:
thanks @ericvn I have addressed this comment.